components/console-backend-service/internal/gqlschema/schema.graphql

```diff
@@ -691,6 +706,8 @@ type Query {
     pod(name: String!, namespace: String!): Pod
     pods(namespace: String!, first: Int, offset: Int): [Pod!]!
 
+    configMap(name: String!, namespace: String!): ConfigMap
```

Comment:
Please add authorization directive (`@HasAccess`) to all new queries/mutations/subscriptions.

Reply:
Directive and acceptance test added :)